smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/DirectedSwiftCodegen.kt

```diff
@@ -90,18 +89,16 @@ class DirectedSwiftCodegen(val context: PluginContext) :
             LOGGER.info("[${service.id}] Generated $numProtocolUnitTestsGenerated tests for protocol ${this.protocol}")
 
             LOGGER.info("[${service.id}] Generating service client for protocol ${this.protocol}")
-
             generateProtocolClient(ctx)
 
             integrations.forEach { it.writeAdditionalFiles(context, ctx, writers) }
-        }
 
-        println("Flushing swift writers")
-        val dependencies = writers.dependencies
-        writers.flushWriters()
+            LOGGER.info("Generating package manifest file")
+            PackageManifestGenerator(ctx).writePackageManifest(writers.dependencies)
 
-        println("Generating package manifest file")
-        writePackageManifest(settings, fileManifest, dependencies, shouldGenerateTestTarget)
+            LOGGER.info("Flushing swift writers")
+            writers.flushWriters()
+        }
```

Comment:
Writing the package manifest used to be done with a separate writer; IDK why.  It is now done with a SwiftWriter from the same context as all other generated files.